.github/workflows/gendoc.yml

```
@@ -29,17 +29,42 @@ jobs:
           
       - name: Generate CLI Documentation
         run: |
+          set -e  # Exit immediately if any command fails
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)